src/MUIDataTableHeadCell.js

```diff
@@ -15,6 +17,9 @@ const defaultHeadCellStyles = {
     zIndex: 100,
     backgroundColor: "#FFF",
   },
+  fixedHeaderAndToolbar: {
```

Comment:
64 pixels is something I pulled out of my developer tools. I couldn't find it anywhere in the codebase. Is there any way we can do this better?